modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -24,7 +24,7 @@ link:{attachmentsdir}/api.raml[api.raml]
 +
 . Check Add APIkit Components. In the RAML File field, browse to and select the downloaded `api.raml` file.
 +
-image::apikit-tutorial-3e0e4.png[height=633,width=600]
+image::apikit-tutorial-3e0e4.png[height=633,width=600][alt=New Mule Project window for project setting to select .raml file.]
```

Comment:
I don't think it's necessary to put the alt=. I think you can just put the text. But it also probably doesn't hurt anything to leave it. You can ask Fer what he thinks. He knows more about asciidoc than I do.

Reply:
Fer included some alt text in this form some days ago.

Reply:
Good question @valkyrie69 
The [asciidoctor documentation](https://docs.asciidoctor.org/asciidoc/latest/macros/images/) says that the alt text can be entered without the `alt=` attribute declaration. 
I'm OK with that. I personally think it's cleaner to declare the attribute, just in case it conflicts with other attributes, but we rarely use any other attributes in the `image::` macro.